.gitlab-ci.yml

```diff
@@ -88,68 +116,152 @@ Base:
           # - aws/rhel-8.4-aarch64
         INTERNAL_NETWORK: ["true"]
 
-OSTree:
+Base-nightly:
   stage: test
-  extends: .terraform
-  variables:
-    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
-  script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/ostree.sh
-  parallel:
-    matrix:
-      - RUNNER:
-          - openstack/fedora-32-x86_64
-          - openstack/fedora-33-x86_64
-          # See COMPOSER-919
-          # - openstack/fedora-34-x86_64
-          - openstack/rhel-8-x86_64
-          - openstack/rhel-8.4-x86_64
-          # do we want centos as well ?
-          # do we want secondary architectures here ??
+  extends: Base
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER == "aws/rhel-8.4-x86_64"'
```

Comment:
Option 1) use a "contains" operator to match everything that contains "rhel" instead of specifying explicitly
Option 2) Redefine the runners explicitly and leave only rhel ones here.

Reply:
ping ^^^

Reply:
Alright so I added a regexp to match any RHEL-8.X-x86_64 runner... I hope that's what we want to achieve here. If we update the runners to 8.5 or switch back to openstack it should still work.